src/components/PeriodDimension/FixedPeriodFilter.js

```diff
@@ -19,15 +20,26 @@ const FixedPeriodFilter = ({
                 onChange={({ selected }) => onSelectPeriodType(selected)}
                 dense
                 selected={currentPeriodType}
+                disabled={
+                    Array.isArray(allowedPeriodTypes) &&
+                    allowedPeriodTypes.length === 1 &&
+                    allowedPeriodTypes[0] === currentPeriodType
+                }
```

Comment:
There's probably a good reason for this, but it's not obvious to me... why is it only disabled when the length is 1?  Maybe it would be good to pull this out into a variable which is descriptively-named?

Reply:
This is a bit of an edge case. If only one period type is allowed, and that one also happens to be selected, there is no point allowing the user to interact with the dropdown since thus user would only be able to chose the option that is already selected. I'll pull this into a variable named `onlyAllowedTypeSelected`